### PR TITLE
Reduces convert_annfile time complexity

### PR DESCRIPTION
**Pull request by trntv**
Hi @chuangzhu,
Thank you for this lib, it saves a lot of time for someone like me who just looking for a tool to convert OID.

This PR reduces convert_annfile time complexity by avoiding nested grouping loops.

### CODE REVIEW

File: oidv6_to_voc/__init__.py [hunk 1]
```diff
@@ -42,23 +42,22 @@ def convert_annfile(annfile: str, desc: str, imgd: str, outd: str):
         p.name.rstrip('.jpg')
         for p in imgp.rglob('*.jpg') if p.is_file()
     }
-    annl = []
 
+    annl: dict[str,  List[AnnotationRow]] = dict()
```
Comment (chuangzhu) [inline oidv6_to_voc/__init__.py]: Python <= 3.7 may not support `dict[...]` typing annotation. Use `Dict` from package `typing` instead.

**Comment by chuangzhu** [comment]
Thanks for the PR. There's a little problem in the type annotation you wrote which I commented above. Could you please fix it?

File: oidv6_to_voc/__init__.py (hunk 1 above)
Comment (trntv) [reply oidv6_to_voc/__init__.py]: Done

**Comment by chuangzhu** [comment]
Looks good to me, thank you!

**Pull request merged**